symplyphysics/laws/hydro/jurin_equation.py

```diff
@@ -0,0 +1,49 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, angle_type)
+
+# Description
+## The Jurin equation determines the height of the liquid rise in the capillaries.
+
+
+## Law: h = 2 * a * cos(theta) / (⍴ * g * r)
+## Where:
+## h is the height of the liquid column
+## a is the coefficient of surface tension of the liquid
+## theta is the angle of liquid wetting of the capillary wall
+## ⍴ is density of liquid
+## g is acceleration of free fall
+## r is capillary radius
+
+
+height = Symbol("height", units.length)
+surface_tension_coefficient = Symbol("surface_tension_coefficient", units.force / units.length)
+angle = Symbol("angle", angle_type)
+density_of_liquid = Symbol("density_of_liquid", units.mass / units.volume)
+radius = Symbol("radius", units.length)
+
+law = Eq(height, (2 * surface_tension_coefficient * cos(angle)) / (density_of_liquid * radius * units.acceleration_due_to_gravity))
+
+## Note
+## The formula is obtained under the assumption that the surface of the meniscus is a sphere, and is applicable if the height of the liquid is raised (lowered) h is much larger than the radius of the capillary r
+## The smaller the radius of the capillary r, the higher the liquid rises in it. The height of the liquid column rises also with an increase in the coefficient of surface tension of the liquid.
+## In the absence of wetting  theta > 90, cos(theta) < 0, and the liquid level in the capillary drops by the value h. With full wetting  theta =0, cos (theta) = 1, and the radius of the meniscus is equal to the radius of the capillary.
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(surface_tension_coefficient_=surface_tension_coefficient, angle_=angle, density_of_liquid_=density_of_liquid, radius_=radius)
+@validate_output(height)
+def calculate_height(surface_tension_coefficient_: Quantity, angle_: Quantity,
+    density_of_liquid_: Quantity, radius_: Quantity) -> Quantity:
+    result_expr = solve(law, height, dict=True)[0][height]
+    result_height = result_expr.subs({
+        surface_tension_coefficient: surface_tension_coefficient_,
+        angle: angle_,
+        density_of_liquid: density_of_liquid_,
+        radius: radius_,
+    })
+    if Quantity(result_height).scale_factor < radius_.scale_factor:
+        raise ValueError("The height must be greater than the radius")
```

Comment:
`greater or equal`
Also add numbers you've got to the exception.

Reply:
What numbers?

Reply:
Scale factors that you're comparing.

Reply:
I don't quite understand what you mean, I'm throwing an exception anyway.

Reply:
Yes, but imagine you're a user. You've got an exception but you do not see actual numbers to compare and you do not know how to fix it. So I suggest to add numbers to exception message.